Core1Platform/AdministratingCore1/02_ManageUsers.md

```diff
@@ -0,0 +1,260 @@
+
+2. Click the ![Create user](../../Assets/Icons/Edit02.png "[Create user]") (Create user) button.   
+  The *Create user* window is opened.    
+    
+    ![Create user](../../Assets/Screenshots/Core1Platform/AdministratingCore1/UserCreate.png "[Create user]")
+
+3. Click the *Select active option* drop-down list to select whether the new user will be able to log in or not. The following options are available: 
+    - **Active, can log in**
+    - **Inactive, user is locked**
+
+4. Click the *Password type* drop-down list to select how the password is to be created and submitted to the user. The following options are available:
+    - **Generate password and send via email**  
+      A password is generated automatically, and it is sent by Actindo to the email address specified below. 
+    - **Enter password and send by email**    
+      You define an individual password, and it is sent by Actindo to the email address specified below. 
+    - **Specify and save password**   
+      You define an individual password and save it. It is not sent via email.  
+    - **Do not change password**   
+      Use this option if you want to change the user's data, but you do not want to edit the password. 
+
+5. If you have not selected the *Generate password and send via email* option in the *Password type option* drop-down list, define the password in the *Set password* field.
+
+6. Enter the username in the *Login* field. This is the name with which the user must log in.  
+
+7. Enter the name and address of the user. Use the *Address* field to enter street and house number.
+
+8. Enter the email address of the new user.   
+  After you have saved the new user, it will be used to send the password to the user.
+
+9. If desired, click the *Role in company* drop-down list. The following options are available:
+    - Entwicklung (Development)
+    - Buchhaltung (Accounting)
+    - Logistik (Logistics)
+    - Artikelanlage (Creating products)
+    - Einkauf (Procurement)
+    - Marktplätze (Market places)
+    - Anderes (Others)
+
+10. Click the [SAVE] button.   
+  The new user is created and, if applicable, informed about username and password by email. 
+  
+
+
+## Assign user to user group
+
+Assigning users to a user group is necessary to activate a user on a current instance. A user that is not assigned any group, is not active in the instance. This means in reverse, you must remove a user from all groups on an instance to deactivate him or her.
+
+## Prerequisite
+
+- You are granted to assign users to groups.
+- The user groups are already created. For detailed information, see [Create user group](../AdministratingCore1/03_ManageGroups.md#create-user-group "Create user group").
+
+## Procedure
+
+*Settings > Users and groups > Tab BENUTZERVERWALTUNG (USER MANAGEMENT)*
+
+![User management](../../Assets/Screenshots/Settings/UsersGroups/UserManagement/UserManagement.png "[User management]")
+
+1. Click the user you want to assign a group.   
```

Comment:
Assign / assign to: Ich würde sagen,  hier ist es mit"to". In diesem Fall hört es sich für mich komisch an. Wir haben auch mal James dazu gefragt und er meinte mit "to". Merriam Webster sagt auch mit "to". 
https://www.merriam-webster.com/dictionary/assign

Reply:
meine Übersetzerinnen bei SAP haben es mir so beigebracht